.travis.yml

```diff
@@ -261,6 +261,8 @@ matrix:
             #- ant $OPTS -f ide/localhistory test
             - ant $OPTS -f ide/lsp.client test
             - ant $OPTS -f ide/notifications test
+            #- ant $OPTS -f ide/notifications.linux test
+            - ant -Dcluster.config=minimal -Djavac.compilerargs=-nowarn -Dbuild.compiler.deprecation=false -Dtest-unit-sys-prop.ignore.random.failures=true -f ide/notifications.linux test
```

Comment:
Was there already some notifications.linux module in the NetBeans repo at some point? Where did the commented-out reference to this module come from?

Reply:
> Was there already some notifications.linux module in the NetBeans repo at some point? 
> Where did the commented-out reference to this module come from?
No, it is only commented for debug propurses.